tests/conftest.py

```diff
@@ -45,26 +46,39 @@ def load_meta(pdir, meta_file):
 
 for ddir in os.listdir(DATA_DIR):
     pdir = DATA_DIR / ddir
-    meta_file = pdir / "meta.yaml"
-    if not pdir.is_dir():
-        continue
-    if not meta_file.is_file():
-        continue
-
-    metadata = load_meta(pdir, meta_file)
-    VTK_FILES.update({metadata["id"]: metadata["attrs"]})
+    for filename in os.listdir(pdir):
+        if filename.startswith("meta") and filename.endswith(".yaml"):
```

Comment:
I took this straight out of #209 but this system looks like it can be improved to limit metadata duplication. I think it would be reasonable to replace the `"path"` scalar with a list (`"paths"`)

Reply:
handled in #221